dubbo-registry/dubbo-registry-api/src/main/java/org/apache/dubbo/registry/integration/RegistryProtocol.java

```diff
@@ -385,13 +383,11 @@ private Cluster getMergeableCluster() {
     }
 
     private URL getRegisteredConsumerUrl(final URL consumerUrl, URL registryUrl) {
-        if (!registryUrl.getParameter(SIMPLE_CONSUMER_CONFIG_KEY, false)) {
+        if (!registryUrl.getParameter(SIMPLIFIED_KEY, false)) {
             return consumerUrl.addParameters(CATEGORY_KEY, CONSUMERS_CATEGORY,
                     CHECK_KEY, String.valueOf(false));
         } else {
-            String[] paramsToRegistry = getParamsToRegistry(DEFAULT_REGISTER_CONSUMER_KEYS,
-                    registryUrl.getParameter(EXTRA_CONSUMER_CONFIG_KEYS_KEY, new String[0]));
-            return URL.valueOf(consumerUrl, paramsToRegistry, null).addParameters(
+            return URL.valueOf(consumerUrl, DEFAULT_REGISTER_CONSUMER_KEYS, null).addParameters(

```

Comment:
@cvictory would it be possible to bit explain it bit here, what we are doing here?

Reply:
@khanimteyaz in `2.7.0` we hope to simplify URL put into the registry center.

Reply:
ok. 👍